sample-authz/src/main/java/net/trajano/ms/example/authz/HazelcastConfiguration.java

```diff
@@ -19,21 +20,35 @@
     @Value("${token.refreshTokenExpiration:3600}")
     private int refreshTokenExpirationInSeconds;
 
+    private static final Logger LOG = LoggerFactory.getLogger(HazelcastConfiguration.class);
+    //
+    //    @Bean
+    //    public HazelcastInstance cacheManager(Config config) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)